bot/convert_currency.go

```diff
@@ -63,12 +77,13 @@ func ConvertMessage(message string) string {
 
 	// check amount was provided
 	if arraySize > 3 {
-		// convert amount to float64
-		amount, err = strconv.ParseFloat(messageArray[3], 64)
+		// Parse amount with support for both comma and period
+		parsedAmount, err := parseAmount(messageArray[3])
```

Comment:
1. your replace is correct... 

My change there is more related to mount the message back. 

25,90 will not work. 

But the answer should have comma back if the configuration say then the rule should be Comma.